documentation/sidebars.js

```diff
@@ -14,8 +14,41 @@ module.exports = {
     {
       type: "doc",
       id: "welcome",
-      label: "Welcome",
+      label: "Getting Started/Welcome",
```

Comment:
If not specifically defined to have the label named "Getting Started/Welcome" I would prefer to call it simply "Welcome" or "Welcome/Getting Started"